File: Cpp/For e While/3.cpp
//Lista 2 do JUDE
//Questão 3

#include <iostream>
using namespace std;

int main() 
{
int T, P, Valor;
cin >> T;
Valor = 0;

    while (P != 0)
        { 
        cin >> P;
        if (P > Valor)
            {
                Valor = P;
            }
        }

if (Valor > T)
{
cout << "ALARME" << endl;
}
else
{
cout << "O Havai pode dormir tranquilo" << endl;
}
return 0;
}

File: Python/For e While/3.py
#Lista 3 do JUDE
#Questão 2

N=int(input())
Resultado=0
Tradicional=0
Geleia=0

for x in range (N):
    Resultado=int(input())
    if Resultado == 10:
        Tradicional += 1
    if Resultado == 11:
        Geleia += 1
if Tradicional > Geleia:
    print('Tradicional')
else:
    print('Geleia')